requirements.txt

```diff
@@ -1,4 +1,2 @@
-django==1.3
+django==1.9
```

Comment:
We'll definitely test it before deploying. Small changes can be done in order to make it work with 1.9. We don't really need new Django version if it really can't work with our gamma.